package.json

```
@@ -28,7 +28,8 @@
     "coverage": "react-scripts test --env=jsdom --coverage",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 9009 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "lint": "./node_modules/.bin/eslint src"
```

don't need to point at the binary directly. just having `eslint src` will work.

Can we also lint the `test` folder? It has simulators and such in

```suggestion
    "lint": "eslint src test"
```